### ISSUE TITLE
re-use running instance of another profile

### ISSUE THREAD

**Issue by nikhilm** [bakulf/switchy#4]
Thanks for switchy!

I have two profiles, default and foobar. I have 2 nightly instances running at once, one for each profile. 'default' is the one that was launched first so that external applications try to use that to open new tabs.
There are certain websites that I want opened in foobar (have company specific authentication details etc.). I have added the rule to switchy. Switchy asks if I want to open the tab in foobar when i open it in default. Everything is great till this point. But Switchy doesn't realize that another instance of firefox is already running with this foobar profile open. So it tries to launch a new instance, but firefox fails with the dialog

```
A copy of Firefox is already open. Only one copy of Firefox can be open at a time.
```

**Comment by bakulf** [comment]
Yeah, I know. Unfortunately the second instance of FF is executed with -no-remote flag and there is no way to send a message to that instance asking to open a second website.
I don't have a good solution for this issue without touching the FF source code.
A real solution is to change FF in order to support multiple 'remote' instances based on the profile names.  But this cannot be done just with an addon.
